addOns/postman/src/test/java/org/zaproxy/addon/postman/PostmanParserUnitTest.java

```diff
@@ -255,4 +267,15 @@ void shouldHandleRequestBodyModes(Body body, String contentType, String stringBo
                 stringBody,
                 new String(httpMessage.getRequestBody().getContent(), StandardCharsets.UTF_8));
     }
+
+    @ParameterizedTest
+    @MethodSource("errorTestData")
+    void shouldGiveErrors(String collectionJson) throws JsonProcessingException {
+        PostmanParser parser = new PostmanParser();
+        List<String> errors = new ArrayList<>();
+
+        parser.getHttpMessages(collectionJson, errors);
+
+        assertEquals(1, errors.size());
```

Comment:
Would be good to also verify the message, or at least setup the tests to verify that the key exists (you can use existing code by calling `mockMessages`).